webpack.config.js

```diff
@@ -52,6 +52,25 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.css$/,
```

Comment:
I stedet for å duplisere kunne man kanskje hatt felles loader config for .css og .less ?